web3/_utils/module_testing/personal_module.py

```diff
@@ -78,19 +84,19 @@ def test_personal_sendTransaction(
         unlockable_account_pw: str,
     ) -> None:
         assert web3.eth.getBalance(unlockable_account_dual_type) > web3.toWei(1, 'ether')
-        txn_params = {
+        txn_params: TxParams = {
             'from': unlockable_account_dual_type,
             'to': unlockable_account_dual_type,
-            'gas': 21000,
-            'value': 1,
+            'gas': Wei(21000),
+            'value': Wei(1),
             'gasPrice': web3.toWei(1, 'gwei'),
         }
         txn_hash = web3.geth.personal.sendTransaction(txn_params, unlockable_account_pw)
         assert txn_hash
         transaction = web3.eth.getTransaction(txn_hash)
 
-        assert is_same_address(transaction['from'], txn_params['from'])
-        assert is_same_address(transaction['to'], txn_params['to'])
+        assert is_same_address(transaction['from'], cast(ChecksumAddress, txn_params['from']))
```

Comment:
I think the better way here is to do `ChecksumAddress(txn_params['from'])` since that will throw an error during runtime.  I assume the `cast` is to get around the need for multiple layers of casting....  I'm inclined to prefer the multiple layers over `cast` in this situation now that I realize that it has runtime implications.

Reply:
Gotcha, I went back to the layered casting for most examples, but since `transaction['from']` (and `['to']`) are typed as `Union[Address, ChecksumAddress, str]` (to allow for ens) - I think we still need to `cast()` in some situations.